mexca/text/transcription.py

```diff
@@ -205,7 +217,24 @@ def get_default_options(language: Optional[str] = None) -> whisper.DecodingOptio
             without_timestamps=False,
             fp16=torch.cuda.is_available()
         )
-
+    
+    @staticmethod
+    def _get_timestamp(word_timestamps : list, idx : int, timestamp_type : Optional[str] = 'start') -> float:
```

Comment:
```suggestion
    def _get_timestamp(word_timestamps : list, idx : int, timestamp_type : str = 'start') -> float:
```
Here, `timestamp_type` is not optional (which implies it can be `None`) because `None` would throw an error in the dictionary lookup.